spiff/models.py

```diff
@@ -18,34 +25,31 @@ def __init__(
         gnn_class: Type[basic_gnn.BasicGNN],
         hidden_size: int,
         num_gnn_layers: int,
-        readout_function: Callable[
-            [torch.Tensor, torch.Tensor | None, int | None], torch.Tensor
-        ],
+        readout_function: Callable[[torch.Tensor, torch.Tensor], torch.Tensor],
         linear_layer_sizes: List[int],
         linear_activation_func: Type[torch.nn.Module],

```

Comment:
Right now there is no way of adjusting parameters of activation function (LeakyReLU slope etc.)